### PR TITLE
Add zap totals to notes

### PR DESCRIPTION
**Pull request by SamSamskies**

### CODE REVIEW

**Comment by SamSamskies** [comment]
subscriptions still a little buggy. i think i'll need upgrade to the latest version of NDK.

**Comment by SamSamskies** [comment]
didn't do the upgrade, but i found a work around to not save wrong values that the subscription sometimes returns.

File: components/NoteActionRow/NoteActionRow.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  useEffect(() => {
+    if (!ndk) {
+      return;
+    }
+
+    const relaySet = createRelaySet(
+      [...defaultRelayUrls, process.env.NEXT_PUBLIC_STEMSTR_RELAY as string],
+      ndk
+    );
```
Comment (michaelhall923) [inline components/NoteActionRow/NoteActionRow.tsx:95]: id think this relay set would be directly accessible from the ndk object somehow

File: ndk/utils.tsx [hunk 2]
```diff
@@ -406,7 +406,11 @@ export const createZapRequest = async ({
     filter: "writable",
   });
```
Comment (michaelhall923) [inline ndk/utils.tsx:407]: could store these with redux

Comment (michaelhall923) [reply ndk/utils.tsx:407]: wtf thats not the code i linked

Comment (michaelhall923) [reply ndk/utils.tsx:407]: was supposed to be userRelayUrls

**Comment by michaelhall923** [comment]
Can we turn the zap button yellow if the user has liked it?

File: ndk/utils.tsx (hunk 2 above)
Comment (SamSamskies) [reply ndk/utils.tsx:407]: i think using the in memory cache would be better than redux for this. i'll look into it, but not sure if we need that optimization right now.

**Comment by SamSamskies** [comment]
> Can we turn the zap button yellow if the user has zapped it?

@rockymedure what you think?

File: components/NoteActionRow/NoteActionRow.tsx (hunk 1 above)
Comment (SamSamskies) [reply components/NoteActionRow/NoteActionRow.tsx:95]: i don't think it is. i can add it to the NDKProvider, but not sure if we'll need this anywhere else.

**Comment by rockymedure** [comment]
I dig it! Yellow is cool

**Comment by SamSamskies** [comment]
> I dig it! Yellow is cool

any specific hex value you want to use?

**Comment by michaelhall923** [comment]
> any specific hex value you want to use?

maybe the one we use in zap notifications? i have it in the theme as orange

**Comment by rockymedure** [comment]
Sgtm!

**Pull request merged**